awx/ui_next/src/screens/Credential/shared/CredentialForm.test.jsx

```diff
@@ -313,36 +413,71 @@ const credentialTypes = [
     },
     injectors: {},
   },
+  {
+    id: 10,
+    type: 'credential_type',
+    url: '/api/v2/credential_types/10/',
+    related: {
+      credentials: '/api/v2/credential_types/10/credentials/',
+      activity_stream: '/api/v2/credential_types/10/activity_stream/',
+    },
+    summary_fields: {
+      user_capabilities: {
+        edit: false,
+        delete: false,
+      },
+    },
+    created: '2020-04-09T19:20:27.090665Z',
+    modified: '2020-04-09T19:21:11.575214Z',
+    name: 'Google Compute Engine',
+    description: '',
+    kind: 'cloud',
+    namespace: 'gce',
+    managed_by_tower: true,
+    inputs: {
+      fields: [
+        {
+          id: 'username',
+          label: 'Service Account Email Address',
+          type: 'string',
+          help_text:
+            'The email address assigned to the Google Compute Engine service account.',
+        },
+        {
+          id: 'project',
+          label: 'Project',
+          type: 'string',
+          help_text:
+            'The Project ID is the GCE assigned identification. It is often constructed as three words or two words followed by a three-digit number. Examples: project-id-000 and another-project-id',
+        },
+        {
+          id: 'ssh_key_data',
+          label: 'RSA Private Key',
+          type: 'string',
+          format: 'ssh_private_key',
+          secret: true,
+          multiline: true,
+          help_text:
+            'Paste the contents of the PEM file associated with the service account email.',
+        },
+      ],
+      required: ['username', 'ssh_key_data'],
+    },
+    injectors: {},
+  },
 ];
```

Comment:
this file is getting a bit big.  if we move all these at the top to data files, I think we'd be able to get this test file down to ~300 lines, which might be a bit better.